docs/API/REST-V2-Usage.md

```diff
@@ -4,6 +4,8 @@ Base URL: `http://<EVERGREEN_HOST>/rest/v2/`
 
 ## General Functionality
 
+See [docs generated from the OpenAPI spec](./OpenAPI.md) for information on specific endpoints.
```

Comment:
nit: The linke can just be `(OpenAPI.md)` since it's in the same directory.

Reply:
No longer relevant since I merged the files.